### PR TITLE
Increase the timeout on apiserver requests to fetch node stats.

### PR DESCRIPTION
**Pull request by vishh**
This is an attempt to fix the test. #3883.

### CODE REVIEW

File: test/e2e/cadvisor.go [hunk 1]
```diff
@@ -44,14 +50,15 @@ func CheckCadvisorHealthOnAllNodes(c *client.Client, timeout time.Duration) {
...
+	retries := maxRetries
+	for {
 		errors = []error{}
 		for _, node := range nodeList.Items {
 			// cadvisor is not accessible directly unless its port (4194 by default) is exposed.
 			// Here, we access '/stats/' REST endpoint on the kubelet which polls cadvisor internally.
 			statsResource := fmt.Sprintf("api/v1beta1/proxy/minions/%s/stats/", node.Name)
 			By(fmt.Sprintf("Querying stats from node %s using url %s", node.Name, statsResource))
-			_, err = c.Get().AbsPath(statsResource).Timeout(1 * time.Second).Do().Raw()
+			_, err = c.Get().AbsPath(statsResource).Timeout(timeout).Do().Raw()
```
Comment (zmerlynn) [inline test/e2e/cadvisor.go]: Am I reading this right? Each timeout is a minute, across 2 nodes, for 10 retries, so you want a budget of 20 minutes for this test to fail?

Comment (vishh) [inline test/e2e/cadvisor.go]: Yes. That is the intent. At this point, I don't even know the reason for the failure. We can dial this back once the test passes successfully.

**Comment by zmerlynn** [comment]
Merging to test hypothesis. @ixdy, you may want to watch Jenkins for a while.

**Comment by zmerlynn** [comment]
(Well, once Travis lets me.)

**Comment by zmerlynn** [comment]
Ignoring Travis to try to make next build. Will watch.

**Pull request merged**